### PR TITLE
Replace travis-ci.org with GitHub Actions

### PR DESCRIPTION
**Pull request by floatingatoll**
The old CI system went away a week or so ago, and per team discussion and in order to unblock #528, we're going to switch this specific repository to GitHub Actions and get CI working again.

Hi, reviewers. This pull request is complicated, as in order to prove that I've repaired the CI system, I also need to incorporate several changes to both the Dockerfile and the CIS code itself.

In this PR, please review _only_ the one unique commit in this PR, 44c79a2, which is not present in any other pull request; it simply removes the Travis CI config and adds the GitHub Actions config. The final push of this pull request will contain only that one unique commit; everything else below is _temporarily_ included from the following pull requests, to demonstrate (as seen below) that CI passes once it's all put together; it's all general repair work that isn't specific to Travis CI or GitHub Actions:

- #534 
- #529 
- #530 

Please do _not_ merge this pull request. I am coordinating several interdependent PRs and will land everything when ready.

### CODE REVIEW

File: .github/workflows/make-test.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: make-test
+
+on: [push]
```
Comment (bkochendorfer) [inline .github/workflows/make-test.yml:3]: Do we want this on `pull_request` too?
```yaml
on: [push, pull_request]
```

**Review by bkochendorfer — COMMENTED** [review]
Looking at https://github.com/mozilla-iam/cis/commit/d2a559e619341a9cdfb55b7370af57d1ddbb55f0 looks good to me. One question as I'm not sure if that is how travis worked

**Comment by floatingatoll** [comment]
Aha, that explains something. Yes, I imagine we do. Thanks!

**Review by bkochendorfer — APPROVED** [review]

**Comment by floatingatoll** [comment]
Alright, I applied the final reviewed commit onto master by itself. Two more PRs to go and we'll have green CI again.

**Pull request merged**